examples/waspello/src/cards/MainPage.jsx

```diff
@@ -243,7 +243,7 @@ const List = ({ list, index, cards }) => {
       index={index}
     >
       {(provided, snapshot) => (
-        <div className='list-wrapper'
+        <div className={`list-wrapper`}
```

Comment:
Could you pls revert this change since it doesn't do anything and it's not what you'd usually write for a React component static `className`.